pykickstart/commands/updates.py

```diff
@@ -71,3 +73,46 @@ def parse(self, args):
             self.url = _ns.updates[0]
 
         return self
+
+
+class F34_Updates(F7_Updates):
+    removedKeywords = F7_Updates.removedKeywords
+    removedAttrs = F7_Updates.removedAttrs
+
+    def __str__(self):
+        retval = KickstartCommand.__str__(self)
+
+        if self.url:
+            retval += "updates %s\n" % self.url
+
+        return retval
+
+    def _getParser(self):
+        op = KSOptionParser(
+            prog="updates", version=F7, description="""
+            Specify the location of an updates.img
+            for use in installation."""
+        )
+        url_action = op.add_argument(
+            "url", metavar="[URL]", version=F7, help="""
+            If present, the URL for an updates image.
+
+            If not present, anaconda will attempt to
```

Comment:
These doc lines don't need to be wrapped quite so short. 80 columns isn't a hard-limit anymore :)